tests/unit/browser/test_caret.py

```diff
@@ -405,3 +395,82 @@ def test_reverse(self, caret, selection):
         caret.reverse_selection()
         caret.move_to_start_of_line()
         selection.check("one two three")
+
+
+class TestLineSelection:
+
+    def test_line_selection_toggle(self, caret, selection):
```

Comment:
nitpick: No need to repeat the `line_selection` part in every test name since they are already in a `TestLineSelection` class.

Reply:
good point :)